translations/other_rules.tex/pl.po

```diff
@@ -99,6 +118,16 @@ msgid ""
 "\\end{itemize}\n"
 "\n"
 msgstr ""
+"\\subsection*{\\hypertarget{Difficulty}{Poziom Trudności}}\n"
+"Podczas przygotowania gry, gracze wybierają Poziom Trudności\\index{Difficulty} gry.\n"
+"Wyróżnia się cztery Poziomy Trudności, każdy z inną Premią Początkową\\index{Starting Bonus}, którą gracze otrzymują podczas kroku 16. przygotowania do gry:\n"
+"\\begin{itemize}\n"
+"  \\item \\textbf{Łatwy} – Rzuć 2 Kośćmi \\includesvg[height=10px]{\\svgs/resource_die.svg} i otrzymaj Zasoby z obydwu – LUB – \\textbf{Przeszukaj} (2) Talię Artefaktów, dwukrotnie.\n"
```

Comment:
Nie chciałem tu przesuwać "Dwukrotnie" przed Przeszukaj, bo mogłoby to być mylące, tj. dwukrotnie w połączeniu z (2) mogłoby być rozumiane jako to samo

Reply:
Imo tak jest dobrze